CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 88 Release Notes
 
+### Removal of Google Genomics Backend
+Code relating to the Google Genomics API has been removed since Google has entirely disabled that service.
+Cloud Life Sciences (deprecated) and Google Batch (recommended) remain the two viable GCP backends. 
```

Comment:
We have a `### Removals` heading at the bottom of this release (and we should do a release soon, there's a lot of stuff)

Reply:
More bikeshedding, I would include the Cromwell names (`v1aplha`, `v2beta`, `batch`) here in addition to the GCP product names, to reduce confusion.